src/screens/FeedsManager/EditSupportedChainDialog.tsx

```diff
@@ -63,7 +62,7 @@ export const EditSupportedChainDialog = ({
 
   const initialValues = {
     chainID: cfg.chainID,
-    chainType: ChainTypes.EVM,
+    chainType: cfg.chainType,
```

Comment:
fixed a bug here when loading an existing chain config , it shows EVM instead of the actual saved chain type.